include/smack/SmackRep.h

```diff
@@ -67,6 +67,10 @@ class SmackRep {
 
   vector<Region> memoryRegions;
   const llvm::DataLayout* targetData;
+
+  // TODO Make this width a parameter to generate bitvector-based code.
+  unsigned width;
```

Comment:
Please correct the name of this field as requested. Either `pointerWidth` or `addressWidth` would be better names.